docs/design_document/design.tex

```diff
@@ -155,8 +160,8 @@ \subsubsection{Results and Data Analysis}
 This project uses DMA to ideally improve the agent's rate of classification over time. Without using DMA, the agent will only remember the information it learned in its last training epoch which most likely will not be as high of classification rate. This gives us something to compare DMA agianst.The results of using a method to mitigate catastrophic interference should be huge in comparison to an online learning technique that does not. We will conduct a quantitative 2-sample data analysis to show a significant increase in the rate of successful classification when using the Dual Memory Architecture.
 
 \section{Conclusion}
+There are two large systems that must be implemented to create an intelligent agent that can correctly recognize objects in its environment, these are the Data Collection phase and the Image Recognition phase. For an agent to be able to recognize and predict things about it's environment, this agent must first be trained. For this project, the agent is trained on images from varying angles of common objects in a lab environment. The agent will be provided with pictures from 5 object classes: mugs, staplers, chairs, books, and screwdrivers. Once several images are gathered and labeled, the agent will train itself on this dataset. This is the first step in actual image recognition \cite{ImgRecog}. The main overarching classifier is trained on images from the 5 object classes. This classifier will output a percentage likelihood for each object class and then will pass the highest class prediction percentage to sub-classifiers that can more accurately classify the object because they specialize in the specific features to that object class. Finally, there is an additional step to image recognition, that when implemented, allows the agent to continually learn new information about these objects over time, this is called Sequential Learning \cite{Kirkpatrick}. A DMA is implemented in order to maximize the amount of new and old information that the intelligent agent can encode, store, and retrieve \cite{McDermott}.
```

Comment:
Change DMA to EWC here too. Also maybe write a sentence about image noise around: "For this project, the agent is trained on images from varying angles of common objects in a lab environment."